src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,825 @@
+```
+
+```bash
+$ kubectl -n nginx-test describe statefulset web
+Name:               web
+Namespace:          nginx-test
+CreationTimestamp:  Mon, 30 Nov 2020 21:53:01 +0200
+Selector:           app.kubernetes.io/instance=nginx-ss,app.kubernetes.io/name=nginx-ss
+Labels:             <none>
+Annotations:        <none>
+Replicas:           2 desired | 2 total
+Update Strategy:    RollingUpdate
+  Partition:        0
+Pods Status:        2 Running / 0 Waiting / 0 Succeeded / 0 Failed
+Pod Template:
+  Labels:  app.kubernetes.io/instance=nginx-ss
+           app.kubernetes.io/name=nginx-ss
+  Containers:
+   nginx-ss:
+    Image:        nginxinc/nginx-unprivileged:stable-alpine
+    Port:         80/TCP
+    Host Port:    0/TCP
+    Environment:  <none>
+    Mounts:       <none>
+  Volumes:        <none>
+Volume Claims:    <none>
+Events:
+  Type    Reason            Age   From                    Message
+  ----    ------            ----  ----                    -------
+  Normal  SuccessfulCreate  94s   statefulset-controller  create Pod web-0 in StatefulSet web successful
+  Normal  SuccessfulCreate  85s   statefulset-controller  create Pod web-1 in StatefulSet web successful
+```
+
+##### 2. Export the Service on **cluster-3**
+
+In order to signify that the Service should be visible and discoverable to other clusters in the cluster set, a `ServiceExport` needs to be
+created. The `subctl export` command can be used to automatically create the required `ServiceExport` object:
+
+```bash
+$ subctl export service --namespace nginx-test nginx-ss
+Service exported successfully
+```
+
+Verify that the `ServiceExport` object has been created for the `nginx-ss` Service within the nginx-test namespace:
+<!-- markdownlint-disable no-trailing-spaces -->
+```bash
+$ kubectl -n nginx-test describe serviceexport nginx-ss
+Name:         nginx-ss
+Namespace:    nginx-test
+Labels:       <none>
+Annotations:  <none>
+API Version:  multicluster.x-k8s.io/v1alpha1
+Kind:         ServiceExport
+Metadata:
+  Creation Timestamp:  2020-11-30T19:59:44Z
+  Generation:          1
+  Resource Version:    83431
+  Self Link:           /apis/multicluster.x-k8s.io/v1alpha1/namespaces/nginx-test/serviceexports/nginx-ss
+  UID:                 2c0d6419-6160-431e-990c-8a9993363b10
+Status:
+  Conditions:
+    Last Transition Time:  2020-11-30T19:59:44Z
+    Message:               Awaiting sync of the ServiceImport to the broker
+    Reason:                AwaitingSync
+    Status:                False
+    Type:                  Valid
+    Last Transition Time:  2020-11-30T19:59:44Z
+    Message:               Service was successfully synced to the broker
+    Reason:                
+    Status:                True
+    Type:                  Valid
+Events:                    <none>
+```
+<!-- markdownlint-enable no-trailing-spaces -->
+When the Service is exported successfully, it can be discovered as `nginx-ss.nginx-test.svc.clusterset.local` across the cluster set.
```

Comment:
```suggestion
Once the Service is exported successfully, it can be discovered as `nginx-ss.nginx-test.svc.clusterset.local` across the cluster set.
```